packages/validators/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@contentful/experiences-validators",
+  "private": true,
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/types.d.ts",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/contentful/experience-builder.git"
+  },
+  "scripts": {
+    "dev": "vite",
+    "clean": "rimraf dist",
+    "prebuild": "npm run clean",
+    "build": "rollup -c rollup.config.mjs",
+    "preview": "vite preview",
+    "test": "vitest",
+    "test:coverage": "vitest run --coverage"
+  },
+  "files": [
+    "dist/**/*.*"
+  ],
+  "typesVersions": {
+    "*": {
+      "types": [
+        "./dist/types.d.ts"
```

```suggestion
        "./dist/index.d.ts"
```
As mentioned above we have the same content already in the `index.d.ts` which is a more appropriate export file for this module.